record-and-playback/presentation/scripts/process/presentation.rb

```diff
@@ -198,12 +202,15 @@
       FileUtils.cp_r("#{pres_dir}/thumbnails", "#{target_pres_dir}/thumbnails")
     end
 
-    BigBlueButton.logger.info("Generating closed captions")
-    ret = BigBlueButton.exec_ret('utils/gen_webvtt', '-i', raw_archive_dir, '-o', target_dir)
-    if ret != 0
-      raise "Generating closed caption files failed"
+    if File.exist?("#{captions_meeting_dir}/captions.json")
+      BigBlueButton.logger.info("Copying closed captions")
+      FileUtils.cp("#{captions_meeting_dir}/captions_playback.json", "#{target_dir}/captions.json")
```

Comment:
This should read the `captions.json` file (since that's the file that gets updated when new captions are uploaded) and convert it to the playback format.